sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -178,7 +178,7 @@ def test_log_base():
     assert log(3**3, 3) == 3
     assert log(5, 1) == zoo
     assert log(1, 1) == nan
-    assert log(Rational(2, 3), 10) == (-log(3) + log(2))/log(10)
+    assert log(Rational(2, 3), 10) == log(S(2)/3)/log(10)
     assert log(Rational(2, 3), Rational(1, 3)) == -log(2)/log(3) + 1
```

Comment:
This (and the next lines) should also be checked locally.

Reply:
@jksuom line 181 is giving `True`
And if line 182 is changed to `log(Rational(2, 3), Rational(1, 3)) == log(S(1)/2)/log(3) + 1`, then also it is giving True